src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,4 +49,16 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code income} for display to the user.
+     */
+    public static String formatIncome(Income income) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(income.getName())
+                .append("; Amount: ")
+                .append(income.getAmount())
+                .append("; Date: ")
+                .append(income.getDateTime());
```

Comment:
I tried doing that but it also becomes quite messy since you are going to hardcode all the "Amount" and "Date". Like a bunch of StringJoiner.add(transaction.getName().toString())